src/generate_supplemental_weekly_plot.R

```diff
@@ -8,14 +8,36 @@ generate_supplemental_weekly_plot <- function(daily_plot_data_arg, df_list_arg,
   week_plot_data <- site_param_df %>%
     filter(DT_round %within% interval(start_date, end_date))
 
-  site_vector <- c("tamasag",
-                    "legacy",
-                    "lincoln",
-                    "timberline",
-                    "prospect",
-                    "boxelder",
-                    "archery",
-                    "river bluffs")
+  #default is lower network
+  site_vector <- c("tamasag", # rist
+                   "legacy",
+                   "lincoln",
+                   "timberline",
+                   "prospect",
+                   "boxelder", # elc
+                   "archery",
+                   "river bluffs")
+
+  if(network == "virridy"){
```

Comment:
Question: where is the `network` object initialized? Is this supposed to be a new function arg? If so I don't see those updates in the function arguments.

Reply:
found it